lib/versionmanager.js

```diff
@@ -322,7 +323,7 @@ function queryVersions(packageMap, options = {}) {
     // validate options.versionTarget
     options.versionTarget = options.versionTarget || 'latest';
 
-    let bar;
+    let bar = {tick: () => {}};
     if (!options.json && options.loglevel !== 'silent') {
         bar = new ProgressBar('[:bar] :percent', {total: packageList.length*2+1, width: 20});
```

Comment:
Can you explain the reasoning behind setting the total to `packageList.length*2+1` instead of just `packageList.length`? With the former, some ticks will be longer than others. With the latter, there will be perhaps an initial pause, but then each tick will be a proportional percentage of the total. We could do a 1 or more ticks for the initial pause, i.e. n + c instead of 2n + 1.

Reply:
Just wanted the progress bar to do something while waiting for the first `package-json` response to finish. But I'm also fine with only tick'ing when responses come in - changed it accordingly. Though, the response ticks are not necessarily a proportional percentage either (in terms of time) since responses can vary in their completion-time. The first tick is so the bar is actually shown.

Reply:
I guess I am unsure what you mean by "first" `package-json` finish. Is `package-json` called more than once for each module? Unless I am missing something I am only seeing the call in `queryVersions`. Thanks!

Reply:
Sorry, what I actually meant is the [first resolving of `getPackageVersion`](https://github.com/tjunnone/npm-check-updates/blob/c96fc0eca5737c8540a7fcbf9359f1e1e9312752/lib/versionmanager.js#L370) (though, the most time inside `getPackageVersion` will probably be spent waiting for the `package-json` response). I'm saying "first" because `queryVersions` [`map`s](https://github.com/tjunnone/npm-check-updates/blob/c96fc0eca5737c8540a7fcbf9359f1e1e9312752/lib/versionmanager.js#L385) the calls to `getPackageVersion` and one of the promises will be the first one to resolve.

With the [`bar.tick` inside the `getPackageVersionProtected`](https://github.com/tjunnone/npm-check-updates/blob/c96fc0eca5737c8540a7fcbf9359f1e1e9312752/lib/versionmanager.js#L362) every `map` call lets the bar progress for every package - and so the bar wouldn't sit at 0% until the first `getPackageVersion` call resolves.